.travis.yml

```diff
@@ -1,16 +0,0 @@
-language: php
```

Comment:
Hey, travis is known by almost everyone on GitHub/in Open Source. I'd rather keep it, because I am not sure about the new one you introduced (which I did not know), and it is always risky to use new tools for this (I like when such things are as boring as possible, so using reliable/known tool seems better to me).